thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/anomaly/detection/trigger/DataAvailabilityTaskScheduler.java

```diff
@@ -119,7 +135,8 @@ public void close() {
   }
 
   private void populateDetectionMapAndDataset2RefreshTimeMap(
-      Map<DetectionConfigDTO, Set<String>> dataset2DetectionMap, Map<String, Long> dataset2RefreshTimeMap) {
+      Map<DetectionConfigDTO, Set<String>> dataset2DetectionMap,
```

Comment:
I have a feeling we are creating too many customized objects. Can we just reuse the DatasetConfigDTO and get rid of DatasetUpdateStatus? The DatasetConfigDTO should contain everything you need.

Reply:
Good suggestion.